docs/literate.jl

```diff
@@ -53,37 +62,32 @@ function preprocess(content)
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
 
-    # adds the current version of the packages
-    append = """
-    # ### Package and system information
-    # #### Package version
-    # ```julia
-    $(chomp(replace(pkg_status, r"^"m => "# ")))
-    # ```
-    # #### Computer information
-    # ```
-    $(chomp(replace(sprint(InteractiveUtils.versioninfo), r"^"m => "# ")))
-    # ```
-    # #### Manifest
-    # To reproduce this notebook's package environment, you can [download the full Manifest.toml]($(MANIFEST_OUT)).
-    """
-    # This regex add "# " at the beginning of each line
-    # chomp removes trailing newlines
+    """ The regex adds "# " at the beginning of each line; chomp removes trailing newlines """
+    literate_format(s) = chomp(replace(s, r"^"m => "# "))
 
-    # The following gives a preview of the manifest
-    # embedded in a "spoiler", but this is unfortunately very buggy
-    # ```@raw html
+    # <details></details> seems to be buggy in the notebook, so is avoided for now
+    info_footer = """
+    # ---
```

Comment:
Some additional heading as currently done:
```suggestion
    # ---
    # ### Package and system information
    #
```